### DISCUSSION TITLE
Unstick top & bottom when using Smaller?

### DISCUSSION THREAD

**Discussion by t4k** [rxhanson/RectanglePro-Community#490] [Q&A]
I like a lot of the predefined widths for my windows, but often I want to reduce the height of them by a smidge so I can use Nudge to manually create a cascade effect. (See also #471.) When I use the Smaller function on a window that is already set using Right Half or Last Two Thirds, for example, only the width changes. I'd really love a way to have the whole window actually get smaller like I expect. Maybe there is one of those command line options to control this?

**Comment by rxhanson** [comment]
Sorry for the delayed response. Try this Terminal command and let me know if it's not what you're looking for:

```
defaults write com.knollsoft.Hookshot curtainChangeSize -int 2
```

Then restart the app after.
